docs/developing/publishing-apps.md

```diff
@@ -0,0 +1,105 @@
+
+You may also need to include `notices` if your app is required to display any third-party copyright notices, for example due to use of third-party open source libraries.
+
+#### categories
+
+You may select the market categories to which your app belongs. You may select multiple, but you may be asked to make changes if the market moderators feel they are inappropriate.
+
+Currently, the following categories are accepted: productivity, communications, social, webPublishing, office, developerTools, science, graphics, media, games, and other.
+
+#### author
+
+The author can be an individual, organization, or even a pseudo-identity representing the app. In order for users to be able to verify the author of a package, the app author can only be identified by PGP key. The Sandstorm team recommends using keybase.io.
+
+* The `upstreamAuthor` is the name of the primary author of the original app. This indicates the author identified by the key ported the app, which was developed by someone else. If the original author is the one publishing the app, do not include this.
+* The `contactEmail` is the address to contact for any issues with this app. This both includes administrative issues with the app market listing as well as end user support requests. It is very important that this email be monitored.
+* The `pgpSignature` field is where you embed a signed ASCII statement verifying that you are the author of this app package.
+
+#### pgpKeyring
+
+This is where you embed a keyring in GPG keyring format containing the public key needed to verify your signature.
+
+#### description
+
+You should embed a description of your app in Github-flavored Markdown. It may not contain HTML or image tags, as you can attach screenshots separately.
+
+#### shortDescription
+
+Include two or three words here that briefly characterize your app.
+
+#### screenshots
+
+You can attach a number of screenshots here. You should specify the height and width of the picture here in "device-independent pixels". You may embed PNGs or JPGs here.
+
+#### changeLog
+
+Here you may embed a log of changes in Github-flavored Markdown. It is recommended to format this with a H1 heading for each release followed by a bullet list of changes.
+
+## Send to the Sandstorm App Market
+
+In order to submit your app to the market, you need to run the following command:
+
+`spk publish mypackage.spk --webkey=https://alpha-api.sandstorm.io#Rs-0TT13YrNSbv7Fiz5K9bBkLaJn3E5TB0PU1GSn1HE`
+
+(Note: The `--webkey` flag will no longer be needed starting with build 0.106, which hasn't been released yet. If you're building Sandstorm from master, you can already skip this flag.)
+
+(Note: If you are developing your app with vagrant-spk, you can publish from inside your Vagrant box by doing `vagrant-spk ssh` and then adding `--keyring /host-dot-sandstorm/sandstorm-keyring` to the above `spk publish` invocation.)
```

Comment:
Oh hey, this is pretty clear.

Maybe we should stick to recommending what already works, which would be to tell all `vagrant-spk` users at the top of this doc to do the commands via `vagrant ssh`.

Then I have less immediately urgent work to do on `vagrant-spk` and people get a working setup.